plugins/mcreator-localization/help/default/biome/generate_mineshafts_mesa.md

```diff
@@ -0,0 +1 @@
+Select this setting to have Mineshafts used in the Mesa biomes in your biome.
```

Comment:
This could be "badlands mineshafts" to use the current biome name (also in other places where it's used such as the GUI)

Reply:
I took Mesa because in Mojmaps, they use MINESHAFT_MESA, so I wanted to be the more accurate possible.

Reply:
Oh ok, though I think it's better to use the current biome name for readable texts

Reply:
I can change the texts, but I don't think it's a problem to keep mesa for keys.

Reply:
Ye that's fine